pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -641,13 +644,36 @@ func (a *Operator) ensureCSVsInNamespaces(csv *v1alpha1.ClusterServiceVersion, o
 	logger := a.logger.WithField("opgroup", operatorGroup.GetName()).WithField("csv", csv.GetName())
 
 	targetCSVs := make(map[string]*v1alpha1.ClusterServiceVersion)
+
+	copyPrototype := v1alpha1.ClusterServiceVersion{
```

Comment:
I don't have a strong preference here, but I think you could calculate the "prototype" in copyableCSVHash. that would make me more confident that we're hashing the same thing here as in the listwatcher